### ISSUE TITLE
ADA: Correct missing Alt Text in Modular templates and published pages

### ISSUE THREAD

**Issue by CharlesChapman** [chapmanu/git2podio#339]
for ADA compliance, need to require altText for images. New modular templates weren't setup to require that field the way the old templates were. Do in two steps:<br /></p><p>Modify all the widget velocity formats to check for blank in altText field. If empty, temporarily output "photo missing alt description".&nbsp; After nightly sitewide publish, scan site to find all live pages with images currently missing altText. Provide a list to web coordinators and get those pages fixed.<br /></p><p>When that is fixed, roll out revision to 2 and 3 Column Modular data definitions so altText field is required. The only one we can't do that on is the Right Column Callout as the image itself is not required and there's no way to make the required altText dependent on the image field being populated. The reason we don't want to do this first is that pages that were published before and altText left blank would, the next time someone submits, flag that empty altText field and as the error message doesn't jump to the field it would be confusing why the field is now flagged, and hard to find in the collapsed sections.<br />
